### ISSUE TITLE
Find a way to handle security rules error from firestore

### ISSUE THREAD

**Issue by ioaNikas** [blockframes/blockframes#1495]
At the moment, using try/catch doesn't catch errors from firestore rules (getting "try" content executed event if we got a permission denied from firestore).

**Comment by lsenta** [comment]
@ioaNikas Could you rephrase this issue or give an exemple of code / behavior?
I don't understand why having the content of a `try` clause run is a problem.

**Comment by Jackseed** [comment]
outdated > closed
